protobuf/build.rs

```diff
@@ -1,124 +0,0 @@
-extern crate curl;
```

Comment:
I don't think you should remove this, this crate has to compile the protobuf project from source in order to build the conformance test runner.  Do the conformance tests pass with this removed?  I'd probably just avoid touching this crate at all in this PR, since it isn't published and is only used for tests.